#here is most sensor data to ctrl use

std_msgs/Header header

dji_onboard_sdk/api_orientation orientation
dji_onboard_sdk/api_angular_v angular_v
dji_onboard_sdk/api_linear_v linear_v
dji_onboard_sdk/api_linear_a linear_a
dji_onboard_sdk/api_gps_data gps_pos
dji_onboard_sdk/api_gimbal gimbal

float32 status
float32 battery
